packages/@aws-cdk/cloud-assembly-schema/scripts/schema.js

```diff
@@ -0,0 +1,129 @@
+const jsonpatch = require('fast-json-patch');
+const TJS = require('typescript-json-schema');
+const path = require('path');
+const semver = require('semver');
+
+function bump() {
+
+  const metadataPath = '../schema/cloud-assembly.metadata.json';
+
+  var metadata = require(metadataPath);
+
+  const oldVersion = metadata.version
+  const newVersion = semver.inc(oldVersion, 'major');
+
+  console.log(`Updating schema version: ${oldVersion} -> ${newVersion}`);
+
+  const patched = jsonpatch.applyPatch(metadata,
```

Comment:
Why complicate??

```js
fs.writeFileSync('cloud-assembly.version.json', JSON.stringify({ version: newVersion }))`
```

Reply:
Hh I had a short lived variant where I had a `hash` field in that file as well, which is also why it was called `metadata.json`. I'll simplify.